README.md

```diff
@@ -1,54 +1,39 @@
-<a href="https://GridTools.github.io/gridtools"><img src="docs/_static/logo.svg"/></a>
+<a href="https://GridTools.github.io/gridtools"><img src="https://gridtools.github.io/gridtools/latest/_static/logo.svg"/></a>
 <br/><br/>
 <a target="_blank" href="https://opensource.org/licenses/BSD-3-Clause">![License: BSD][BSD.License]</a>
 
-The GridTools framework is a set of libraries and utilities to develop performance portable applications in the area of weather and climate. To achieve the goal of performance portability, the user-code is written in a generic form which is then optimized for a given architecture at compile-time. The core of GridTools is the stencil composition module which implements a DSL embedded in C++ for stencils and stencil-like patterns. Further, GridTools provides modules for halo exchanges, boundary conditions, data management and bindings to C and Fortran.
+**cpp_bindgen** is a library to generate C and Fortran bindings for C++ functions from C++. It is part of the GridTools framework, a set of libraries and utilities to develop performance portable applications in the area of weather and climate.
 
-GridTools is successfully used to accelerate the dynamical core of the [COSMO model](http://cosmo-model.org/) with improved performance on CUDA-GPUs compared to the current official version, demonstrating production quality and feature-completeness of the library for models on lat-lon grids.
-
-Although GridTools was developed for weather and climate applications it might be applicable for other domains with a focus on stencil-like computations.
-
-A detailed introduction can be found in the [documentation](https://GridTools.github.io/gridtools).
+Currently the documentation of **cpp_bindgen** is part of the [GridTools documentation](https://gridtools.github.io/gridtools/latest/user_manual/user_manual.html#interfacing-to-other-programming-languages). For simple examples, see the examples folder of this repository. More advanced examples are available in https://github.com/GridTools/gridtools.
 
 ### Installation instructions
 
+You can easily integrate **cpp_bindgen** in your CMake project with the following snippet which makes the function `cpp_bindgen_add_library()` available.
+
+```cmake
+include(FetchContent)
+FetchContent_Declare(
+  c_bindings
+  GIT_REPOSITORY https://github.com/GridTools/c_bindings.git
+  GIT_TAG        master # consider replacing master by a tagged version
+)
+FetchContent_GetProperties(c_bindings)
+if(NOT c_bindings_POPULATED)
+  FetchContent_Populate(c_bindings)
+  add_subdirectory(${c_bindings_SOURCE_DIR} ${c_bindings_BINARY_DIR})
+endif()
```

Comment:
should we mention FetchContent_MakeAvailable again? In my opinion that should be the preferred way as long as no old CMake versions need to be supported.

Reply:
Let's keep it for now. I think anyway it make sense to update our CMake at some point, then we can also fix it here.